tests/testthat/test_OptimInstance.R

```diff
@@ -78,3 +78,24 @@ test_that("Terminator assertions work", {
   terminator$properties = "multi-crit"
   expect_error(MAKE_INST(terminator = terminator))
 })
+
+test_that("objective_function works", {
+  terminator = terminator = term("evals", n_evals = 100)
+  inst = MAKE_INST_1D(terminator = terminator)
+  expect_numeric(inst$objective_lower(), names = "named", len = inst$search_space$length)
```

Comment:
as mentioned above. `$objective_lower` and `$objective_upper` are really not that necessary right?